qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,108 @@
+Feature: Sending CV
+
+Backgound: 
+	Given user accessed Quantori Career Portal from his browser
+	And user scrolled down to CV sending form
+
+Scenario: User can successfully send CV
```

Comment:
For each scenario, you need to specify its number. For example, 1.2.1_01 - the first scenario, 1.2.1_02 - the second scenario etc.

Reply:
ok, I'll add!